data_structures/k_tree/k_tree.py

```diff
@@ -79,3 +79,30 @@ def _walk(nodes):
 
         if self.root:
             _walk([self.root])
+
+    def print_level_order(self):
+        count = 1
+        newcount = 0
+        stringthing = ''
+
+        def _walk(nodes):
+            nonlocal count, newcount, stringthing
+            new = []
+            if count > 0:
+                for node in nodes:
+                    stringthing += str(node.val)
+                    count -= 1
+                    for child in node.children:
+                        newcount += 1
+                        new.append(child)
+            stringthing += '\n'
+            count = newcount
+            newcount = 0
+
+            if len(new):
```

Comment:
You do not need to explicitly check the length of a list in a conditional. An empty list is falsey and a filled list is truthy.